legal-api/migrations/versions/1ba027b074a5_add_idp_userid_to_users.py

```diff
@@ -0,0 +1,32 @@
+"""add_idp_userid_to_users
+
+Revision ID: 1ba027b074a5
+Revises: e10f724fad6d
+Create Date: 2022-11-22 12:44:50.751121
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '1ba027b074a5'
+down_revision = 'e10f724fad6d'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('users', sa.Column('idp_userid', sa.String()))
+    #op.drop_constraint('users_sub_key', 'users', type_='unique')
```

Comment:
Any reason for the commented out code?

Reply:
This is just the first step. The constraint may stop the data loading step on the next.